_data/communication.yml

```diff
@@ -1,4 +1,11 @@
 websites:
+    - name: Action Network
+      url: https://actionnetwork.org/
+      img: actionnetwork.png
+      tfa: Yes
+      software: Yes
+      doc: https://help.actionnetwork.org/hc/en-us/articles/217206826
```

Comment:
Sure. After you create an account you can edit your profile and enroll in 2FA: 

https://www.evernote.com/l/AD_LlFRUKd1D9qT4EwzyKP-bCbcmrpCodPk

From then on, when you log in, you will be prompted for your 2-factor token:

https://www.evernote.com/l/AD_tGk5KcPxDfZ0RljWNWeooxW4xdE8SOWE

You can edit your 2-factor settings from your profile page:

https://www.evernote.com/l/AD8j0Sk0YmZC5IAW9F7O1Zp5_sAxH7HVa2U